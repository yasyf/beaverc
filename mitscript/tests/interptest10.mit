manycars = fun(n ){
  carmap = {n:n;};
  print("gets here");
  print(carmap);
  n = n - 1;
};

cars = manycars(10);

print("hello");