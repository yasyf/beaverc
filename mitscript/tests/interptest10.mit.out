gets here
{n:10 }
hello
